plugins/remoteaccess/RemoteAccessWidget.cpp

```diff
@@ -252,13 +254,23 @@ RemoteAccessWidget::RemoteAccessWidget( const ComputerControlInterface::Pointer&
 	m_vncView( new VncViewWidget( computerControlInterface->computer().hostAddress(), -1, this, VncView::RemoteControlMode ) ),
 	m_toolBar( new RemoteAccessWidgetToolBar( this, startViewOnly, showViewOnlyToggleButton ) )
 {
+	const auto openOnMasterScreen = VeyonCore::config().ShowFeatureWindowsOnSameScreen();
+	auto master = VeyonCore::instance()->findChild<VeyonMasterInterface *>();
+	const auto masterWindow = master->mainWindow();
+	if( master && openOnMasterScreen )
+	{
+		move( masterWindow->x(), masterWindow->y() );
+	} else {
+		move( 0, 0 );
+	}
+
 	updateRemoteAccessTitle();
 	connect( m_computerControlInterface.data(), &ComputerControlInterface::userChanged, this, &RemoteAccessWidget::updateRemoteAccessTitle );
 
 	setWindowIcon( QPixmap( QStringLiteral(":/remoteaccess/kmag.png") ) );
 	setAttribute( Qt::WA_DeleteOnClose, true );
 
-	m_vncView->move( 0, 0 );
+	m_vncView->move( x(), y() );
```

Comment:
Is this really necessary? The VncView widget is a children of RemoteAccessWidget and thus should be placed at (0, 0) since coordinates are always relative to the parent widget and not absolute.

Reply:
I'll try it again to be sure.

Reply:
You were right. :)